apps/docs/docs/ref/self-hosting-functions/introduction.mdx

```diff
@@ -62,6 +62,30 @@ To try it yourself,
 
 You can view the logs for the Edge Runtime, by visiting Fly.io’s Dashboard > Your App > Metrics. Also, you can serve edge-runtime from multiple regions by running `fly regions add [REGION]`.
 
+## Configuration of SECRET_KEY_BASE
+
+When self-hosting Supabase, it's important to properly configure the SECRET_KEY_BASE parameter. This value is essential for securing your Supabase instance. Below, we outline how to set the value and provide guidance on best practices.
+
+### Purpose of SECRET_KEY_BASE
+
+SECRET_KEY_BASE is a crucial component of Supabase's security infrastructure. It is used for cryptographic functions and securing various aspects of your deployment, including session management and CSRF protection.
```

Comment:
Is this true? My understanding is secret_key_base is only used in realtime rather than the entire supabase stack.